bc_obps/reporting/models/report_new_entrant_emissions.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from registration.models.time_stamped_model import TimeStampedModel
+from reporting.models import EmissionCategory
+from reporting.models.report_new_entrant import ReportNewEntrant
+
+
+class ReportNewEntrantEmissions(TimeStampedModel):
```

Comment:
This should be singular, it's been the convention for all the other models
`ReportNewEntrantEmission`